### PR TITLE
Create github endpoint

### LINKED ISSUE #1: Create a set of API queries to get the data from Github REST API
This issue involves identifying which Github queries might be relevant to the project MVP and get them on one of the pages. I will work on this after going through a crash course on React.

### PR DESCRIPTION
**Pull request by prakharrathi25**
## What this PR Does

This PR adds a github endpoint which is connected to the Github API and gets a list of repositories and other details that are present in a user's profile. 

## Changes Made 
- Created a `dashboard.tsx` file for the stars count 
- Created a `/api/github.js` file for the api endpoint

image

image

## Blockers

Facing this one TypeScript Error @Green-Ranger11 Not sure how to fix it
image

This closes #1

### CODE REVIEW

**Comment by prakharrathi25** [comment]
Thanks @Green-Ranger11 for fixing the typescript issues in my PR.

File: gitdash/components/team.tsx [hunk 1]
```diff
@@ -116,8 +116,7 @@ export default function TeamSection() {
             <TeamContent>
               <TeamHeading>Backend Web Developer</TeamHeading>
               <TeamText>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Auctor
-                neque sed imperdiet nibh lectus feugiat nunc sem.
+                I mostly break stuff, sometimes it turns out to be something cool.
```
Comment (anandrajaram21) [inline gitdash/components/team.tsx:119]: Can you maybe make this a bit longer?

File: gitdash/pages/api/github.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const { Octokit } = require('@octokit/rest')
```
Comment (anandrajaram21) [inline gitdash/pages/api/github.js]: Can you rename this file to github.ts to maintain uniformity?

File: gitdash/yarn.lock [hunk 3]
```diff
@@ -0,0 +1,4792 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (anandrajaram21) [inline gitdash/yarn.lock]: Can you also please remove the yarn.lock file?

**Review by anandrajaram21 — CHANGES_REQUESTED** [review]
1. Can you make your bio section on the home page a bit longer?
2. Can you rename the file github.js to github.ts?
3. Can you also please remove the yarn.lock file too?

File: gitdash/components/team.tsx (hunk 1 above)
Comment (prakharrathi25) [reply gitdash/components/team.tsx:119]: Yup, sure. I was trying to be quirky here. I will add a few more lines to it.

File: gitdash/pages/api/github.js (hunk 2 above)
Comment (prakharrathi25) [reply gitdash/pages/api/github.js]: Renamed the file to github.ts

File: gitdash/yarn.lock (hunk 3 above)
Comment (prakharrathi25) [reply gitdash/yarn.lock]: Deleted the yarn.lock file. Sorry about the yarn and npm issue. I will use npm commands in the future.

**Comment by prakharrathi25** [comment]
I am not able to figure out why the deploy fails when it works on our systems. @robzwolf any way we can fix this?

**Review by anandrajaram21 — APPROVED** [review]
Awesome, these changes are perfect

**Comment by prakharrathi25** [comment]
> Awesome, these changes are perfect

Thanks for your valuable comments. 👍🏼

**Pull request merged**